### PR TITLE
keep the dataset's color_map same as  color_map in  visualize.py

### PR DESCRIPTION
**Pull request by BeyondYourself**
keep the dataset's color_map same as  color_map in  visualize.py

### CODE REVIEW

**Comment by BeyondYourself** [comment]
@wuyefeilin 大佬看看，这样应该没问题了吧

File: tools/gray2pseudo_color.py [hunk 1]
```diff
@@ -52,7 +55,8 @@ def get_color_map_list(num_classes):
             color_map[i * 3 + 2] |= (((lab >> 2) & 1) << (7 - j))
             j += 1
             lab >>= 3
-
+    # color_map = [color_map[i:i + 3] for i in range(0, len(color_map), 3)]
```
Comment (wuyefeilin) [inline tools/gray2pseudo_color.py]: 注释去掉一下就行

**Review by wuyefeilin — COMMENTED** [review]
注释删除一下

File: tools/gray2pseudo_color.py (hunk 1 above)
Comment (BeyondYourself) [reply tools/gray2pseudo_color.py]: 好的

**Comment by BeyondYourself** [comment]
@wuyefeilin 已更正

**Comment by LutaoChu** [comment]
LGTM

**Pull request merged**